airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/property_history.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": ["null", "object"],
+    "properties": {
+      "value": {
+        "type": ["null", "string"]
+      },
+      "source-type": {
+        "type": ["null", "string"]
+      },
+      "source-id": {
```

Comment:
same question as [here](https://github.com/airbytehq/airbyte/pull/7787/files#r765338260)

Reply:
A Property can either be changed by workflow, form or manully. The id for either is saved in the `source-id` . The thin is, that for workflow the id is really a id as in integer. but the id of a form is a mixture of characters and numbers. Therefore it is safer to just cast this as a string.